core/block_svg.js

```diff
@@ -639,6 +639,20 @@ Blockly.BlockSvg.prototype.onMouseDown_ = function(e) {
     // Click, not drag, so stop waiting for other touches from this identifier.
     Blockly.Touch.clearTouchIdentifier();
   } else if (!this.isMovable()) {
+    // Click handler for immovable blocks.
+    var mouseupWrapper = Blockly.bindEvent_(document,
```

Comment:
Ah right, bindEvent_ is correct.

Yes, I can take this over.

I'm familiar with blockly's touch strategy, and I also have had a hard time figuring this one out :/